climada/engine/impact.py

```diff
@@ -390,6 +388,42 @@ def impact_per_year(self, all_years=True, year_range=None):
             year_set[year] = sum(self.at_event[orig_year == year])
         return year_set
 
+    def impact_at_reg(self, exposures):
+        """Aggregate impact at the region_id level (if these are specified in exposure)
+        or at the admin 0 level (if they are not). This method works only if Impact.imp_mat
+        was stored during the impact calculation.
```

Comment:
The `exposures` object must be the very same Exposures that was given as argument to `ImpactCalc`, right?
That is a very tight constraint but not verified.
Makes me feel uneasy. I'd be happier if the argument wasn't an `Exposures` object but just an arbitrary aggregation column. So you would call it like this:

```python
impact.impact_at_reg(exposures.gdf.region_id)
```

instead of 

```python
impact.impact_at_reg(exposures)
```

This would transfer all the responsibility for being consistent to the caller.

In any case, the docstring could be more insisting about consistency and perhaps even provide an example.

Reply:
it now works with user-defined aggregation regions. If `None`, `admin_0` levels are used.